decisions/2020/10/28-frontend-initiatives.md

```diff
@@ -0,0 +1,102 @@
+**Date**: 2020-10-28
+
+**Attending (11)**: Austin, Viktor, Birk, Christos, Edoardo, Hendrik,
+Jan Henrik,  Jan-Gerke, Joe, Martin, Milagros, Wahed
+
+**Next Meeting**: Wednesday 11 November, 2020 @ 15:00
+
+## Agenda
+
+-	Quick general updates:
+
+	-	New unified GitHub workflows for [apps](https://github.com/dhis2/workflows/blob/master/ci/dhis2-verify-app.yml), [libs](https://github.com/dhis2/workflows/blob/master/ci/dhis2-verify-lib.yml), [node](https://github.com/dhis2/workflows/blob/master/ci/dhis2-verify-node.yml)
+
+	-	Work started on practical plans on GitHub to JIRA migration (#135)
+
+- 	Updates from active working groups
+
+-	New working group for Feature toggling
+
+	-	All teams do an initial check on "known" features to toggle
+	-	Working group extracts use cases and works on a proof-of-concept
+	-	Pilot integration into chosen apps
+
+-	Cypress and Gherkhin - next steps for decision
+
+-	Migrating d2-ui components
+
+-	Tech debt review
+
+## Links
+
+(DHIS2 account required)
+
+*	[Agenda](https://github.com/dhis2/notes/issues/134)
+
+## Notes
+
+### General updates
+
+Let Viktor know if you are having issues with the unified workflows.
+
+### Working groups
+
+Four working groups in May:
+
+*	_Independent app releases_: good progress
+
+*	_PWA offline support_: put off to after 2.35
+
+*	_Platform improvements_: main one with progress, converted several apps to
+	the platform, progressive modernization working well, moving away from d2
+
+	Some challenges related to d2, and replacing d2-ui components, gradual
+	migration may be possible with the d2 runtime adapter. Hard to replace when
+	using the d2 models.
+
+	Testing and automation, more getting involved.
+
+	Porting d2-ui to ui.
```

Comment:
Could you elaborate on this a bit more? I'm not quite sure how to interpret these 2 points.

Reply:
Alright, expanded a bit on the context. Does that help?